crowbar_framework/app/views/nodes/_buttons_power.html.haml

```diff
@@ -2,23 +2,28 @@
   = link_to t(".identify"), hit_node_path(@node.handle, "identify"), class: "btn btn-default", remote: true
 
 - unless @node.admin?
-  - if @node[:platform_family] != 'windows' || @node.bmc_set?
-    %button.btn.btn-default.dropdown-toggle{ type: "button", data: { toggle: "dropdown" } }
-      = t(".actions")
-      %span.caret
+  - if @node[:platform] != 'windows' || @node.bmc_set?
 
-    %ul.dropdown-menu{ role: "menu" }
-      - if @node[:platform_family] != 'windows'
+    .dropdown
+      %button.btn.btn-default.dropdown-toggle{ "aria-expanded" => "true", "aria-haspopup" => "true",
+        "data-toggle" => "dropdown", :type => "button", role: "menu" }
+        = t(".actions")
+        %span.caret
+      %ul.dropdown-menu{ "aria-labelledby" => "dropdownMenu1" }
         %li{ role: "presentation" }
-          = link_to t(".reboot"), hit_node_path(@node.handle, "reboot"), role: "menuitem", remote: true
+          %a{ :href => hit_node_path(@node.handle, "reboot"), role: "menuitem",
+            data: { blockui_click: t(".blockui_message") } }
+            = t(".reboot")
         %li{ role: "presentation" }
-          = link_to t(".shutdown"), hit_node_path(@node.handle, "shutdown"), role: "menuitem", remote: true
-        %li.divider{ role: "presentation" }
-      - if @node.bmc_set?
```

Comment:
I think you've lost this condition. In the old code, Power On button could be shown only `if @node.bmc_set?`